tribe-ext-export-mec-events.php

```diff
@@ -381,34 +411,33 @@ public function venues_csv_setup() {
 		/**
 		 * Generate the CSV files.
 		 *
+		 * @since TBD Fixed warning from order of header output.
+		 *
 		 * @param string $csv_file_name - The name of the CSV file to be created
-		 * @param array  $header        - The name of the columns
-		 * @param array  $data
+		 * @param array  $csv_header    The name of the columns
+		 * @param array  $data          Event entries.
 		 */
-		public function generate_csv( $csv_file_name, $header, $data ) {
-
+		public function generate_csv( $csv_file_name, $csv_header, $data ) {
 			$fh = fopen( 'php://output', 'w' );
 
-			/**
-			 * Write the file header for correct encoding ( UTF8 )
-			 */
-			fprintf( $fh, chr( 0xEF ) . chr( 0xBB ) . chr( 0xBF ) );
-
 			header( 'Cache-Control: must-revalidate, post-check=0, pre-check=0' );
 			header( 'Content-Description: File Transfer' );
 			header( 'Content-type: text/csv' );
 			header( "Content-Disposition: attachment; filename={$csv_file_name}" );
 			header( 'Expires: 0' );
 			header( 'Pragma: public' );
 
-			fputcsv( $fh, $header );
+			// Write the file header for correct encoding ( UTF8 ).
+			fprintf( $fh, chr( 0xEF ) . chr( 0xBB ) . chr( 0xBF ) );
 
+			// Output CSV data.
+			fputcsv( $fh, $csv_header );
```

Comment:
```suggestion
			fputcsv( $fh, $csv_header );
			
```
I added a line break